nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -49,6 +49,7 @@ BtNavigator::BtNavigator()
     "nav2_reinitialize_global_localization_service_bt_node",
     "nav2_rate_controller_bt_node",
     "nav2_distance_controller_bt_node",
+    "nav2_speed_controller_bt_node",
```

Comment:
Please submit a PR to add this PR to migration guide + this BT node to the plugins list.